rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -258,3 +263,83 @@ def test_eigenlayer_balances(
         amount=FVal('0.114063122816914142'),
         usd_value=FVal('0.1710946842253712130'),
     )
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [[
+    '0x328fE286b2bE1895959E2601364AfE9DAA22aba3',
+    '0x4ebE485C1DF060f6Fc6E3C3b200EBc21Fe11a94D',
+]])
+@pytest.mark.parametrize('arbitrum_one_manager_connect_at_start', [(get_arbitrum_allthatnode(weight=ONE, owned=True),)])  # noqa: E501
+@pytest.mark.parametrize('mocked_current_prices', [{
+    'eip155:42161/erc20:0x2f2a2543B76A4166549F7aaB2e75Bef0aefC5B0f': FVal(69928),  # wBTC
+    'eip155:42161/erc20:0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1': FVal(1.001),  # dai
+    'eip155:42161/erc20:0x82aF49447D8a07e3bd95BD0d56f35241523fBab1': FVal(3583.17),  # weth
+}])
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+def test_gmx_balances(
+        arbitrum_one_inquirer: 'ArbitrumOneInquirer',
+        arbitrum_one_transaction_decoder: 'OptimismTransactionDecoder',
+        arbitrum_one_accounts: list[ChecksumEvmAddress],
+        inquirer: 'Inquirer',  # pylint: disable=unused-argument
+) -> None:
+    """
+    Test querying balances for GMX. We use an address with 2 different positions and the other
+    one has a single position. There are both shorts and longs in the data.
+    """
+    database = arbitrum_one_transaction_decoder.database
+    for tx_hash in (
+        '0x195cdb1bde8da223c7e6216166f7e8b5b79fee6409dde145a8f3e95223ebcc51',  # short btc
+        '0x59d136f8d1b366a895ddf4136fb45d62b862e79f4a937f5a9b33ceec47eaf32f',  # long btc
+        '0xf14d5a3f3fa4a4c324f95b70db81502f3d7c1910782381ed82f3580da64b093f',  # long eth
+    ):
+        tx_hex = deserialize_evm_tx_hash(tx_hash)
+        get_decoded_events_of_transaction(
+            evm_inquirer=arbitrum_one_inquirer,
+            database=arbitrum_one_transaction_decoder.database,
```

Comment:
Why create a `database` variable? And then not use it here?

Probably just fine to have `arbitrum_one_transaction_decoder.database` in both and save one line.

Whatever you do choose one and stick to it. Don't use different var here and different in `GmxBalances`

Reply:
I deleted it after the suggestion from oj but had to restore and I did loose the change FFFFFFFFF